Documentation/Storage-Configuration/Object-Storage-RGW/ceph-object-swift.md

```diff
@@ -0,0 +1,140 @@
+---
+title: Object Store with Keystone and Swift
+---
+
+You can configure the Swift API and the Keystone integration of Ceph RGW natively via the Object Store CRD, which allows native integration of the Rook operated Ceph RGW into [OpenStack](https://www.openstack.org/) clouds.
```

Comment:
This is a little pedantic, but when we had the Ceph foundation's doc expert review our docs, the biggest suggestion he gave was for us to avoid use of "you," "our," "we," etc.

He suggested that we use "imperative" language in our docs. So this doc line should be shifted around to read something like this: 

"Ceph RGW can integrate natively with the Swift API and Keystone via the CephObjectStore CRD. This allows native integration of Rook-operated Ceph RGWs into [OpenStack](link)"

Please adjust the rest of the document as well. Doing a file search for "you" is usually sufficient to find which lines should be adjusted.

Reply:
fa106d4e19f711f3493b7d8f34791f46dc51e4cc

This is not pedantic but good advice! Thanks. I rephrased the document accordingly.
I would suggest to add a section on the imperative language in the Contributing/documentation.md file.